database/seeders/BetaSeeder.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Seeder;
+
+class BetaSeeder extends Seeder
```

Comment:
How this seeder will be run on the server?
Manually? Via post deploy actions?

Actually developers don't have access to the container with this project. 
There is no workflow to run commands in the container on the server.

Reply:
I don't know how we want to make this - maybe manually running is enough? It's just seed of two accounts to admin panel - nothing complicated and when we once create accounts, we don't have to seed it all the time.

Reply:
Sure, it can be done manually. In the future it can be adjusted if needed.